docs/rfcs/merger_architecture.md

```diff
@@ -71,29 +71,31 @@ This is our current schema:
 This is best illustrated with an example.
 Suppose we have the following graph:
 
-![](relater_example1.png)
+![](matcher_example1.png)
 
 We have two existing works: ABC and DEF.
 We receive an update to B telling us it now has edges B→A and B→D.
 
 0.  The existing DB is as follows:
 
         src   | tail_v | component_v | is_redirect | redirected_target
-        A     | B      | ABC         | -           | -
-        B     | A      | ABC         | -           | -
-        C     | B      | ABC         | -           | -
-        D     | F      | DEF         | -           | -
-        E     | D      | DEF         | -           | -
-        F     | -      | DEF         | -           | -
+        A     | B      | ABC         | true        | ABC
```

Comment:
I took the liberty to modify some of the things that I think we agreed on a whiteboard session after this document was created, but feel free to scratch that if it doesn't make sense